src/index.ts

```diff
@@ -0,0 +1,1003 @@
+import * as async from "async";
+import { BN, rlp } from "ethereumjs-util";
+import Common from "ethereumjs-common";
+import * as util from "util";
+import DBManager from "./dbManager";
+import {
+  bodyKey,
+  bufBE8,
+  hashToNumberKey,
+  headBlockKey,
+  headHeaderKey,
+  headerKey,
+  numberToHashKey,
+  tdKey
+} from "./util";
+
+const Block = require("ethereumjs-block");
+const Ethash = require("ethashjs");
+const Stoplight = require("flow-stoplight");
+const level = require("level-mem");
+const semaphore = require("semaphore");
+
+export default class Blockchain {
+  _common: any;
+  _genesis: any;
+  _headBlock: any;
+  _headHeader: any;
+  _heads: any;
+  _initDone: boolean;
+  _initLock: any;
+  _putSemaphore: any;
+  _staleHeadBlock: any;
+  _staleHeads: any;
```

Comment:
Nice resorting and - generally - property collection, finally gives some overview what's inside. 😄  👍